### ISSUE TITLE
Regression: playback failure on GLK

### ISSUE THREAD

**Issue by keqiaozhang** [thesofproject/linux#687]
This issue is caused by a change in topology thesofproject/sof/pull/1047, but kernel has not been updated accordingly, may need to backport thesofproject/linux/pull/675 to topic/cros-sof-v4.14-rebase branch.

How to reproduce:
```
aplay -Dhw:0,0 -f dat -c 2 audio.wav
aplay: set_params:1403: Unable to install hw params:
ACCESS:  RW_INTERLEAVED
FORMAT:  S16_LE
SUBFORMAT:  STD
SAMPLE_BITS: 16
FRAME_BITS: 32
CHANNELS: 2
RATE: 48000
PERIOD_TIME: 84000
PERIOD_SIZE: 4032
PERIOD_BYTES: 16128
PERIODS: (4 5)
BUFFER_TIME: 340000
BUFFER_SIZE: 16320
BUFFER_BYTES: 65280
TICK_TIME: 0
```
Dmesg:
```
[   46.845151] sof-audio-pci 0000:00:0e.0: period_bytes:0xfc0
[   46.845152] sof-audio-pci 0000:00:0e.0: periods:16
[   46.845161] sof-audio-pci 0000:00:0e.0: stream_tag 1
[   46.845197] sof-audio-pci 0000:00:0e.0: ipc tx: 0x60010000
[   46.845370] sof-audio-pci 0000:00:0e.0: error: ipc error for 0x60010000 size 0x14
[   46.845395] sof-audio-pci 0000:00:0e.0: error: hw params ipc failed for stream 1
[   46.845403] sof-audio-pci 0000:00:0e.0: ASoC: 0000:00:0e.0 hw params failed: -19
[   46.845407]  Speakers: ASoC: hw_params FE failed -19
[   46.845413] sof-audio-pci 0000:00:0e.0: pcm: free stream 0 dir 0
[   46.845446] sof-audio-pci 0000:00:0e.0: ipc tx: 0x60030000
[   46.845581] sof-audio-pci 0000:00:0e.0: error: ipc error for 0x60030000 size 0xc
[   46.858219] sof-audio-pci 0000:00:0e.0: pcm: close stream 0 dir 0
```

Trace:
   ```
 0      2          IPC             70822.291667       198.541672    src/ipc/handler.c:643    ipc: pm -> restore
    0      2          IPC             82248.750000     11426.458008    src/ipc/handler.c:239    ipc: comp -1 -> params
    0      1          IPC             82265.989583        17.239584    src/ipc/handler.c:244    ipc: comp -1 not found
    0      2          IPC             82495.625000       229.635422    src/ipc/handler.c:367    ipc: comp -1 -> free
    0      1          IPC             67690.052083              nan    src/ipc/handler.c:372    ipc: comp -1 not found
```

Test recipe:
kernel-cros-sof-v4.14-rebase:164bdbb81
firmware/tplg-master:24e241b

**Comment by ranj063** [comment]
@keqiaozhang sorry that took a while to fix. I've updated the branch now. Could you please give it a go and let me know if the issue still persists?

**Comment by keqiaozhang** [comment]
Thanks @ranj063 .
Confirmed that this issue is fixed on cros-sof-v4.14-rebase branch.
Commit ID: b178744d9
